examples/with-realm-web/.env

```diff
@@ -0,0 +1 @@
+REALM_APP_ID=realm-example-bspbt
```

Comment:
If this is a secret please use `.env.local` instead, and in that case create a `.env.local.example` that can be used to create it, because `.env.local` will be ignored by git

Reply:
It's not a secret since the request URL contains this App ID too